.github/workflows/template.yml

```diff
@@ -89,36 +111,8 @@ jobs:
         run: |
           while read -r cmd
           do
-            eval $cmd
+            eval sudo $cmd
```

Comment:
Why is `sudo` necessary -- we're `root` in GitHub Actions.

Reply:
not sure, but without sudo it doesn't work :(
https://github.com/carpentries/styles/runs/992384878#step:14:12